xbmc/ServiceManager.cpp

```diff
@@ -176,11 +180,15 @@ bool CServiceManager::InitStageTwo(const CAppParamParser &params)
   m_fileExtensionProvider.reset(new CFileExtensionProvider(*m_addonMgr,
                                                            *m_binaryAddonManager));
 
-  m_powerManager.reset(new CPowerManager());
+  m_powerManager.reset(new CPowerManager(*m_announcementManager,
```

Comment:
this hard dependencies are really bad and make things worse. every service should be stoppable without the need to know what other component may depend on it.